src/editor.js

```diff
@@ -1628,10 +1627,7 @@ export class OdooEditor extends EventTarget {
         const selection = this.document.defaultView.getSelection();
         this._updateToolbar(!selection.isCollapsed);
 
-        if (this._currentMouseState === 'mousedown') {
-            // _fixFontAwesomeSelection will be called when the mouseup event is triggered.
-            this._selectionChanged = true;
-        } else {
```

Comment:
Pourquoi t'as supprimé ça ? C'était utilisé par @sebgeelen  le fixFontAwesomeSelection

Reply:
`this._selectionChanged` était toujours true, du coup ça ne servait à rien.

Reply:
c'est que qqch avait cassé le comportement entre temps parce que normalement `this._selectionChanged` ne devrait pas toujours être à true

Reply:
Je viens de chercher un peu, je pense que dans [mon commit qui introduit cette fonctionalité](https://github.com/odoo-dev/odoo-editor/commit/bc6a890f5ac7d1f567d99d8fe0674e64c445ddea), j'ai oublié de reseté la variable `selectionChanged` à `false` après avoir fixé la fontawesome selection. ( dans la méthode `_fixFontAwesomeSelection`).

My bad du coup.

Donc ou tu fixe ça dans cette PR @Goaman, ou je fait une PR à part.
Mais il ne faut pas enlevé cette fonctionnalité.

Reply:
Pourquoi on ne peut pas "fixer" à chaque mouseup ?

Reply:
on pourrait en effet